max_pos_0_int: 1.0

kp_pos_0: {x: 2.9, y: 2.9, z: 14}
ki_pos_0: {x: 0.002, y: 0.002, z: 0.003}
kd_pos_0: {x: 1.1, y: 1.1, z: 3.2}

k_R_0: {x: 0.0, y: 0.005, z: 0.005}
k_Omega_0: {x: 0.0, y: 0.001, z: 0.001}

kp_q: 22
ki_q: 0.1
max_e_q_int: 1.0
kd_q: 2.4

tau_q_i: 0.015
tau_q_i_dot: 0.015
tau_w_i: 0.015
